keyboards/synthandkeys/bento_box/info.json

```diff
@@ -0,0 +1,64 @@
+{
+    "manufacturer": "Synth-and-Keys",
+    "keyboard_name": "bento_box",
+    "maintainer": "Synth-and-Keys",
+    "bootloader": "stm32-dfu",
+    "bootmagic": {
+        "matrix": [2, 0]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B2","pin_b": "B1"},
+            {"pin_a": "A13","pin_b": "A14"},
+            {"pin_a": "A15","pin_b": "B3"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["A9", "A8", "B9", "B6", "B0", "A5"],
+            ["A2", "A7", "A4", "A1", "B8", "B5"],
+            ["A6", "A3", "A0", "B7", "B4", null]
+        ]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 3], "x": 3.5, "y": 0 },
+                { "matrix": [0, 4], "x": 4.5, "y": 0 },
+                { "matrix": [1, 1], "x": 1, "y": 0.5 },
+                { "matrix": [1, 3], "x": 3.5, "y": 1 },
+                { "matrix": [1, 4], "x": 4.5, "y": 1 },
+                { "matrix": [2, 0], "x": 0, "y": 2 },
+                { "matrix": [2, 1], "x": 1, "y": 2 },
+                { "matrix": [2, 2], "x": 2, "y": 2 },
+                { "matrix": [3, 0], "x": 0, "y": 3 },
+                { "matrix": [3, 1], "x": 1, "y": 3 },
+                { "matrix": [3, 2], "x": 2, "y": 3 },
+                { "matrix": [3, 3], "x": 3.5, "y": 2.5 },
+                { "matrix": [3, 4], "x": 4.5, "y": 2.5 },
+                { "matrix": [4, 0], "x": 0, "y": 4 },
+                { "matrix": [4, 1], "x": 1, "y": 4 },
+                { "matrix": [4, 2], "x": 2, "y": 4 },
+                { "matrix": [4, 3], "x": 3.5, "y": 3.5, "h": 1.5},
+                { "matrix": [4, 4], "x": 4.5, "y": 3.5, "h": 1.5}
```

Comment:
```suggestion
                { "matrix": [0, 0], "x": 3.5, "y": 0 },
                { "matrix": [0, 1], "x": 4.5, "y": 0 },

                { "matrix": [0, 2], "x": 1, "y": 0.5 },
                { "matrix": [0, 3], "x": 3.5, "y": 1 },
                { "matrix": [0, 4], "x": 4.5, "y": 1 },

                { "matrix": [0, 5], "x": 0, "y": 2 },
                { "matrix": [1, 0], "x": 1, "y": 2 },
                { "matrix": [1, 1], "x": 2, "y": 2 },

                { "matrix": [1, 2], "x": 0, "y": 3 },
                { "matrix": [1, 3], "x": 1, "y": 3 },
                { "matrix": [1, 4], "x": 2, "y": 3 },
                { "matrix": [1, 5], "x": 3.5, "y": 2.5 },
                { "matrix": [2, 0], "x": 4.5, "y": 2.5 },

                { "matrix": [2, 1], "x": 0, "y": 4 },
                { "matrix": [2, 2], "x": 1, "y": 4 },
                { "matrix": [2, 3], "x": 2, "y": 4 },
                { "matrix": [2, 4], "x": 3.5, "y": 3.5, "h": 1.5},
                { "matrix": [2, 5], "x": 4.5, "y": 3.5, "h": 1.5}
```
This layout doesn't make sense; there are 18 keys but the direct_pins configuration above only specifies 17 pins (hence the single `null` pin). Please double check this.